### PR TITLE
sql: add logic and parallel test files to static-tests package

### LINKED ISSUE #5339: stress: failed test in cockroach/sql/sql.test: TestLogic
Binary: cockroach/static-tests.tar.gz sha: https://github.com/cockroachdb/cockroach/commits/1df05137c01a75933b462ee717e9e035afde5b55

Stress build found a failed test:

```
=== RUN   TestLogic
--- FAIL: TestLogic (0.00s)
	logic_test.go:753: No testfiles found (globs: [testdata/[^.]*])
```

Run Details:

```
0 runs so far, 0 failures, over 5s
0 runs so far, 0 failures, over 10s
1 runs completed, 1 failures, over 12s
FAIL
```
Please assign, take a look and update the issue accordingly.

### PR DESCRIPTION
**Pull request by RaduBerinde**
The static test package did not include the sql test files. Until recently the
tests just silently did not run anything (now they fail). This change copies the
test files in the staging directory for the package (as part of `make
testbuildall`). The `stress.sh` script is also fixed to run each test binary
from the respective package directory.

Tested by running `build/build-static-binaries.sh`, unpacking the bundle in a
temp dir, copying stress, and running `cloud/aws/stress.sh`.

Fixes #5339.

---

This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/5358)

### CODE REVIEW

**Comment by RaduBerinde** [comment]
@mberhault

**Comment by mberhault** [comment]
LGTM

File: sql/parallel_test.go [hunk 1]
```diff
@@ -160,10 +160,14 @@ func (t *parallelTest) setup() {
...
 	defer leaktest.AfterTest(t)()
 
-	paths, err := filepath.Glob(*paralleltestdata)
+	glob := string(*paralleltestdata)
+	paths, err := filepath.Glob(glob)
 	if err != nil {
 		t.Fatal(err)
 	}
+	if len(paths) == 0 {
+		t.Fatalf("No testfiles found (glob: %s)", glob)
```
Comment (tschottdorf) [inline sql/parallel_test.go]: don't regular logic tests have the same issue?

**Comment by RaduBerinde** [comment]
*[sql/parallel_test.go, line 169 \[r3\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/5358#-KD5Gbp1wtzXlwJazu1c-r3-169:-KDA7YA1VwKtnaBYSAz5:388038744) ([raw file](https://github.com/cockroachdb/cockroach/blob/6c586dccbadc845eb286bcec789cfd7d3cdc41bb/sql/parallel_test.go#L169)):*
Yeah, I already did that and forgot about htis one.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/5358)*

**Pull request merged**